### ISSUE TITLE
renderLineHighlight with accessibility mode

### ISSUE THREAD

**Issue by galyech** [microsoft/monaco-editor#494]
monaco-editor version: 0.9.0
Browser: Chrome
OS: Windows
Steps or JS usage snippet reproducing the issue:

The line highlight is disabled when accessibility is on:
renderLineHighlight: (accessibilityIsOn ? "none" : opts.viewInfo.renderLineHighlight),

I would like to use both of them at the same time,
Is there a reason for that?

**Comment by alexandrudima** [comment]
`accessibilitySupport: "on"` means that the editor becomes optimized for usage by users with a Screen Reader. This flag should be set to `on` if:

a) you are running in an environment where you can detect a Screen Reader is attached (e.g. Electron or an embedded WebBrowser control in a desktop application)
b) you have a user facing setting where users can choose to turn on special Screen Reader support (e.g. Google Docs)

If a) or b) do not apply to your use case, then leave the `accessibilitySupport` to `"auto"`. Screen Readers work correctly under `"auto"`, with the exception of e.g. wrapped lines, because there is no ARIA way of expressing wrapped lines to Screen Readers.

From our documentation: https://github.com/Microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators
<img width="712" alt="screen shot 2017-07-06 at 08 36 07" src="https://user-images.githubusercontent.com/5047891/27898415-3c076734-6226-11e7-9b3b-7db571f95d94.png">

**Comment by alexdima** [comment]
This is now obsolete, the line highlight is rendered even when accessibility support is on.
